tests/flags/flags.go

```diff
@@ -65,6 +66,7 @@ func init() {
 	flag.StringVar(&PreviousReleaseRegistry, "previous-release-registry", "index.docker.io/kubevirt", "Set registry of the release to test updating from")
 	flag.StringVar(&ConfigFile, "config", "tests/default-config.json", "Path to a JSON formatted file from which the test suite will load its configuration. The path may be absolute or relative; relative paths start at the current working directory.")
 	flag.BoolVar(&SkipShasumCheck, "skip-shasums-check", false, "Skip tests with sha sums.")
+	flag.BoolVar(&IsDualStackedNetworkConfig, "dual-stack", false, "Run network related tests in dual stack mode")
```

Comment:
The flag was going ot be the opposite, `--skip-dual-sack-test`. and will skip that test, this way if users forget the flag the worst that can happend is that it's using a single stuck cluster and has one test failing, but if forget the flag and it's dual-stack it will do the proper testing.